tests/drivers/i2s/i2s_speed/src/test_i2s_speed.c

```diff
@@ -45,7 +45,12 @@ static int16_t data_r[SAMPLE_NO] = {
 
 #define BLOCK_SIZE (2 * sizeof(data_l))
 
-K_MEM_SLAB_DEFINE(rx_0_mem_slab, BLOCK_SIZE, NUM_BLOCKS, 32);
+/*
+ * The NXP I2S driver queues 2 receive buffers to avoid receive overflows
+ * Allocate an extra block so we do not see test failures due to
+ * allocate failures
```

Comment:
Let's maybe change the comment to something like:

"NUM_BLOCKS is the number of blocks used by the test. Some of the drivers, e.g. i2s_mcux_flexcomm, permanently keep ownership of a few RX buffers. Add a few more RX blocks to satisfy this requirement."

We could also reserve 2 extra buffers instead of 1, not to change this again when another I2S driver which requires more buffers is added to the tree.

Reply:
Done, thank you for the suggestion